README.md

```diff
@@ -1,6 +1,7 @@
 [![Build status](https://img.shields.io/travis/stryker-mutator/stryker4s/master.svg)](https://travis-ci.org/stryker-mutator/stryker4s)
 [![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker4s?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![BCH compliance](https://bettercodehub.com/edge/badge/stryker-mutator/stryker4s?branch=master)](https://bettercodehub.com/)
+[![Maven Central](https://img.shields.io/maven-central/v/io.stryker-mutator/stryker4s-core_2.12.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:io.stryker-mutator)
```

Comment:
I know it's almost like nitpicking but, could we move the Maven central badge next to the build badge.
And can we update the badge so it's green, it feels weird to have it in orange. 

(https://img.shields.io/maven-central/v/io.stryker-mutator/stryker4s-core_2.12.svg?colorB=fedcba?label=Maven%20Central)